content/en/docs/armory-agent/agent-monitoring.md

```diff
@@ -9,6 +9,22 @@ description: >
 
 {{< include "early-access-feature.html" >}}
 
+## Available metrics
+
+| Name | Description |
+|-----------|---------------|
+|`container_cpu_usage_seconds_total` |  Container CPU usage |
+|`container_memory_working_set_bytes` |  Container working set memory usage |
```

Comment:
Remove `container_cpu_usage_seconds_total` and `container_memory_working_set_bytes`, these metrics are not returned by agent but may be available from prometheus in general.
Also add these ones
```
go_memstats_alloc_bytes  {Desc: Number of bytes allocated and still in use. Type: gauge}
go_memstats_alloc_bytes_total  {Desc: Total number of bytes allocated, even if freed. Type: counter}
go_memstats_buck_hash_sys_bytes {Desc: Number of bytes used by the profiling bucket hash table. Type: gauge}
go_memstats_frees_total {Desc: Total number of frees. Type: counter}
go_memstats_gc_cpu_fraction {Desc: The fraction of this program's available CPU time used by the GC since the program started. Type: gauge}
go_memstats_gc_sys_bytes {Desc: Number of bytes used for garbage collection system metadata. Type: gauge}
go_memstats_heap_alloc_bytes {Desc: Number of heap bytes allocated and still in use. Type: gauge}
go_memstats_heap_idle_bytes {Desc: Number of heap bytes waiting to be used. Type: gauge}
go_memstats_heap_inuse_bytes {Desc: Number of heap bytes that are in use. Type: gauge}
go_memstats_heap_objects {Desc: Number of allocated objects. Type: gauge}
go_memstats_heap_released_bytes  {Desc: Number of heap bytes released to OS. Type: gauge}
go_memstats_heap_sys_bytes {Desc: Number of heap bytes obtained from system. Type: gauge}
go_memstats_last_gc_time_seconds {Desc: Number of seconds since 1970 of last garbage collection. Type: gauge}
go_memstats_lookups_total {Desc: Total number of pointer lookups. Type: counter}
go_memstats_mallocs_total {Desc: Total number of mallocs. Type: counter}
go_memstats_next_gc_bytes {Desc: Number of heap bytes when next garbage collection will take place. Type: gauge}
go_memstats_sys_bytes {Desc: Number of bytes obtained from system. Type: gauge}

process_cpu_seconds_total {Desc: Total user and system CPU time spent in seconds. Type: counter}
process_max_fds {Desc: Maximum number of open file descriptors. Type: gauge}
process_open_fds {Desc: Number of open file descriptors. Type: gauge}
process_resident_memory_bytes  {Desc: Resident memory size in bytes. Type: gauge}
process_start_time_seconds {Desc: Start time of the process since unix epoch in seconds. Type: gauge}
process_virtual_memory_bytes {Desc: Virtual memory size in bytes. Type: gauge}
process_virtual_memory_max_bytes {Desc: Maximum amount of virtual memory available in bytes. Type: gauge}